azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/implementation/FunctionDeploymentSlotImpl.java

```diff
@@ -72,4 +74,15 @@ public Completable zipDeployAsync(File zipFile) {
             return Completable.error(e);
         }
     }
-}
\ No newline at end of file
+
+    @Override
+    Observable<SiteInner> submitSite(final SiteInner site) {
+        return submitSiteWithoutSiteConfig(site);
```

Comment:
the webapp base has `site.withSiteConfig(new SiteConfig());`. Does it need to be added here?

Reply:
Service will return this, if set...

```
Function App app settings are not expected to be empty. Please try again with WEBSITE_CONTENTAZUREFILECONNECTIONSTRING as a storage account connection string.: Function App app settings are not expected to be empty. Please try again with WEBSITE_CONTENTAZUREFILECONNECTIONSTRING as a storage account connection string.
```